functorch/csrc/dim/dim.cpp

```diff
@@ -867,7 +867,7 @@ mpy::object Tensor::from_positional(Arena & A, at::Tensor tensor, Slice<DimEntry
     }
     AT_ASSERT(last == 0 || last == -1);
     if (!seen_dims) {
-        return mpy::object::steal(THPVariable_Wrap(std::move(tensor)));
+        return mpy::object::steal(THPVariable_Wrap(tensor));
```

Comment:
Ugh, we do not have the clang-tidy rule enabled for this? Or is it just that it's not flagging except for newer Clang-Tidy?

Reply:
Detected from LSAN

Reply:
I meant for detecting the slicing. Why is this flagged in our current clang-tidy config via [cppcoreguidelines-slicing](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/slicing.html#cppcoreguidelines-slicing). Is it because clang-tidy is to old?

Reply:
From lsan

Reply:
@Skylion007 Sometimes due to merge race conditions, clang-tidy may not have chance to check them.

Reply:
Clang-tidy runs on all files though, not just changed ones though so that shouldn't affect it?